CHANGELOG.md

```diff
@@ -4,13 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
```

Comment:
I would leave the unreleased. Otherwise it will get removed and added over and over again. This has potential for merge conflicts.

Reply:
I fear that would look quite confusing, don't you think:

## Unreleased

## v0.1.1 - 2020-05-08
### Added
- Docs from the SDD
- Doc generator from CRDs
- GitRepo file templates
- Add an empty file for each

Reply:
🤦 End user perspective vs developer perspective. 🤷.